scripts/ValidateProjectVersionUpdated.ps1

```diff
@@ -56,7 +56,7 @@ Catch {
     Exit 1
 }
 
-$currentPublishedVersion = [System.Management.Automation.SemanticVersion]$nugetIndex.items[0].upper
+$currentPublishedVersion = [System.Management.Automation.SemanticVersion]$nugetIndex.items[$nugetIndex.items.Count - 1].upper
```

Comment:
did the ordering from the service change under us? shouldn't we do some client side sorting first to ensure versions are in the right order?

Reply:
Ordering did not change. Items are just split up in different groups. Will update PR to reflect feedback in kiota PR in a bit.